lib/htmltoword/document.rb

```diff
@@ -145,7 +143,7 @@ def inject_image_content_types(source)
 
       #inject missing extensions into document
       missing_exts.each do |ext|
-        doc.at_css("Types").add_child( "<Default Extension='#{ext}' ContentType='image/#{ext}'/>")
+        doc.at_css("Types").add_child( "<Default Extension='#{ext}' ContentType='image/#{content_type_from_extension(ext)}'/>")
```

Comment:
Here is where I'm determining the content type from the file extension.  The file extension and the content type need to stay separate otherwise you run into issues with a jpg file not rendering in the document because the "jpg" extension isn't registered in the content types.